### DISCUSSION TITLE
where should i get the animated icon for github readme

### DISCUSSION THREAD

**Discussion by ghost** [community/community#129326] [New to GitHub]
i want to add animated emoji or icon to my github readme

**Comment by iydebu** [comment]
go to https://www.flaticon.com/animated-icons site u can download animated gif from there 
then remove the background of the gif from the free tool https://onlinegiftools.com/remove-gif-background 
then upload it to the https://imgbb.com/ it will host your gif for free then copy the direct link
and use it on readme

**Comment by andialfiann** [comment]
great
